### ISSUE TITLE
version 1.14 generates errors for tests with CTest fixture

### ISSUE THREAD

**Issue by ebouteillon** [microsoft/vscode-cmake-tools#3124]
### Brief Issue Summary

Tests with fixture are reported as *failed* in "Testing" tab, even if they are *Passed* in output logs. Failure indicated is "Expecting one test result.". No issue with previous version 1.13.

Minimal sample of `CMakeLists.txt` to reproduce. test_1, test_2 and test_3 should be reported as "passed" instead of "failed".

```cmake
CMAKE_MINIMUM_REQUIRED (VERSION 3.7)

project(cmake_fixture)

enable_testing()

# Create a fixture to setup before tests / cleanup after tests
add_test(NAME pre_ctest COMMAND sleep 5)
add_test(NAME post_ctest COMMAND sleep 1)
set_tests_properties(pre_ctest PROPERTIES FIXTURES_SETUP database)
set_tests_properties(post_ctest PROPERTIES FIXTURES_CLEANUP database)
set_tests_properties(pre_ctest post_ctest PROPERTIES RESOURCE_LOCK database_access)

add_test(NAME test_1 COMMAND sleep 1)
set_tests_properties(test_1 PROPERTIES FIXTURES_REQUIRED database)
set_tests_properties(test_1 PROPERTIES RESOURCE_LOCK database_access)

add_test(NAME test_2 COMMAND sleep 1)
set_tests_properties(test_2 PROPERTIES FIXTURES_REQUIRED database)
set_tests_properties(test_2 PROPERTIES RESOURCE_LOCK database_access)

add_test(NAME test_3 COMMAND sleep 1)
set_tests_properties(test_3 PROPERTIES FIXTURES_REQUIRED database)
set_tests_properties(test_3 PROPERTIES RESOURCE_LOCK database_access)
```

### CMake Tools Diagnostics

```shell
{
  "os": "linux",
  "vscodeVersion": "1.77.3",
  "cmtVersion": "1.14.30",
  "configurations": [
    {
      "folder": "/home/axis/workspace/buggy_cmake",
      "cmakeVersion": "3.26.3",
      "configured": true,
      "generator": "Unix Makefiles",
      "usesPresets": false,
      "compilers": {
        "C": "/opt/rh/devtoolset-8/root/usr/bin/gcc",
        "CXX": "/opt/rh/devtoolset-8/root/usr/bin/g++"
      }
    }
  ],
  "cpptoolsIntegration": {
    "isReady": true,
    "hasCodeModel": false,
    "activeBuildType": "Debug",
    "buildTypesSeen": [
      "Debug"
    ],
    "requests": [],
    "responses": [],
    "partialMatches": [],
    "targetCount": 0,
    "executablesCount": 0,
    "librariesCount": 0,
    "targets": []
  },
  "settings": [
    {
      "communicationMode": "automatic",
      "useCMakePresets": "auto",
      "configureOnOpen": true
    }
  ]
}
```

### Additional Information

Screenshot below shows in "output log" pane that *test_3* is Passed, but it is reported as a failure in "Testing" pane and an error is displayed.

![image](https://user-images.githubusercontent.com/32395889/233666188-bbe6fb8d-3ca9-4bd4-b3fa-a9e371847048.png)

**Comment by benmcmorran** [comment]
The fix for this isssue from #3160 is now publicly available in the pre-release version of the extension 1.15.6.

**Comment by ebouteillon** [comment]
Pre release 1.15.6 fixed it. Tanks for the fix.
